instrumentserver/gui/instruments.py

```diff
@@ -188,7 +216,7 @@ def makeRemoveWidget(self, fullName: str):
         w.pressed.connect(lambda: self.removeParameter(fullName))
         return w
 
-    def removeParameter(self, fullName: str):
+    def removeParameter(self, fullName: str, deleteServerSide: Optional[bool] = True):
         items = self.plist.findItems(
```

Comment:
Is there a situation where you would not want to delete the server side parameter?

Reply:
This happens when a second GUI gets a 'parameter-deletion' broadcast and needs to only delete the GUI elements.